src/Dialogs/UnsafePasteDialog.vala

```diff
@@ -28,11 +29,10 @@ public class Terminal.UnsafePasteDialog : Granite.MessageDialog {
 
     construct {
         image_icon = new ThemedIcon ("dialog-warning");
-        primary_text = _("This command is asking for Administrative access to your computer");
 
         secondary_text =
-            _("Copying commands into the Terminal can be dangerous.") + " " +
-            _("Be sure you understand what each part of this command does.");
+            _("Copying commands into Terminal can be dangerous.") + " " +
+            _("Be sure you understand what each part of the pasted text does before continuing.");
```

Comment:
If we're changing these anyways, we should probably remove the forced space and make this a single translatable string
```suggestion
            _("Copying commands into Terminal can be dangerous. Be sure you understand what each part of the pasted text does before continuing.");
```

Reply:
This was separated in master and I have no idea why, so I left it as-is. But I can combine them.